snap/HOWTO.md

```diff
@@ -1,109 +1,107 @@
 ```
 Now we have a functioning three node cluster. 
 
-# An Example Database
+## An Example Database
 
 Let's create an example database ...
+```bash
+  $ curl -X PUT http://admin:password@10.210.199.199:5984/example
+  $ curl -X PUT http://admin:password@10.210.199.199:5984/example/aaa -d '{"test":1}' -H "Content-Type: application/json"
+  $ curl -X PUT http://admin:password@10.210.199.199:5984/example/aab -d '{"test":2}' -H "Content-Type: application/json"
+  $ curl -X PUT http://admin:password@10.210.199.199:5984/example/aac -d '{"test":3}' -H "Content-Type: application/json"
 ```
-curl -X PUT http://admin:password@10.210.199.199:5984/example
-curl -X PUT http://admin:password@10.210.199.199:5984/example/aaa -d '{"test":1}' -H "Content-Type: application/json"
-curl -X PUT http://admin:password@10.210.199.199:5984/example/aab -d '{"test":2}' -H "Content-Type: application/json"
-curl -X PUT http://admin:password@10.210.199.199:5984/example/aac -d '{"test":3}' -H "Content-Type: application/json"
-```
-... And see that it is sync'd accross the three nodes.
-```
-curl -X GET http://admin:password@10.210.199.199:5984/example/_all_docs
-curl -X GET http://admin:password@10.210.199.254:5984/example/_all_docs
-curl -X GET http://admin:password@10.210.199.24:5984/example/_all_docs
+... And see that it is created on all three nodes.
+```bash
+  $ curl -X GET http://admin:password@10.210.199.199:5984/example/_all_docs
+  $ curl -X GET http://admin:password@10.210.199.254:5984/example/_all_docs
+  $ curl -X GET http://admin:password@10.210.199.24:5984/example/_all_docs
 ```
-# Backing Up CouchDB
+## Backing Up CouchDB
 
-Our back up server is on 10.210.199.242. We will manually replicate this from one (anyone) of the nodes.
+Our backup server is on 10.210.199.242. We will manually replicate to this from one (can be any one) of the nodes.
+```bash
+  $ curl -X POST http://admin:Be1stDB@10.210.199.242:5984/_replicate -d '{"source":"http://10.210.199.199:5984/example", "target":"example", "continuous":false,"create_target":true}' -H "Content-Type: application/json"
+  $ curl -X GET http://admin:Be1stDB@10.210.199.242:5984/example/_all_docs
 ```
-curl -X POST http://admin:password@10.210.199.242:5984/_replicate -d '{"source":"http://10.210.199.199:5984/example", "target":"example", "continuous":false,"create_target":true}' -H "Content-Type: application/json"
-curl -X GET http://admin:password@10.210.199.242:5984/example/_all_docs
+Whereas the data store for the clusters nodes is sharded:
+```bash
+  $ lxc exec couchdb-c1 ls /var/snap/couchdb/common/data/shards/
 ```
-The data store for the clusters nodes are sharded 
-```
-lxc exec couchdb-c1 ls /var/snap/couchdb/common/2.x/data/shards/
-```
-
-The backup database is a single file.
-```
-lxc exec cdb-backup ls /var/snap/couchdb/common/2.x/data/shards/00000000-ffffffff/
+The backup database is a single file:
+```bash
+  $ lxc exec cdb-backup ls /var/snap/couchdb/common/data/shards/00000000-ffffffff/
```

Comment:
This should only occur if you've set `q=1` on the backup server. I don't see you setting that anywhere in the walkthrough.

It would be better to change this to just show that the backup database is in a single directory, and then `ls /var/snap/couchdb/common/data/shards` rather than giving people the (bad) advice that all backup servers should store a database in a single shard.